errors/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,6 @@
     <application>
         <activity
             android:name=".ErrorActivity"
-            android:theme="@style/Theme.MaterialComponents" />
+            android:theme="@style/Theme.MaterialComponents.NoActionBar" />
```

Comment:
Using an theme with an action bar caused `ErrorActivityTest` to fail as `ErrorActivity` calls `setSupportActionBar`.